federatedscope/autotune/algos.py

```diff
@@ -78,6 +79,8 @@ def get_scheduler(init_cfg):
     #     scheduler = PBT(init_cfg)
     elif init_cfg.hpo.scheduler.startswith('wrap'):
         scheduler = SHAWrapFedex(init_cfg)
+    elif init_cfg.hpo.scheduler == 'fts':
+        scheduler = FTSAlgo(init_cfg)
```

Comment:
No definition of `FTSAlgo`.

Reply:
I just deleted everything about FTSAlgo. We don't need this algorithm now. All process of fts are contained in FTSServer and FTSClient